demo/test-button.html

```diff
@@ -25,7 +25,7 @@
   </style>
   <template>
     <paper-icon-button id="m" icon="menu" alt="menu"></paper-icon-button>
-    <paper-tooltip for="m" margin-top="8">hot dogs</paper-tooltip>
+    <paper-tooltip for="m" offset="8">hot dogs</paper-tooltip>
```

Comment:
No, it's an offset in the direction of the tooltip. So by default, the tooltip is "below", so this is a vertical offset (tooltip is horizontally centered).

If the tooltip is to the left, then this is a horizontal offset (and the tooltip is vertically centered).

I'm drawing the line at not allowing both :)